CHANGELOG.md

```diff
@@ -15,6 +15,13 @@ Our versioning strategy is as follows:
 
 * Upgrade cloudsdk to 0.5 ([#2060](https://github.com/Sitecore/jss/pull/2060)):
   * This upgrade doesn't introduce any breaking changes, however you will have to upgrade your cloudsdk dependencies to meet peer dependencies requirements
+* `[sitecore-jss]` `[sitecore-jss-react]` `[sitecore-jss-nextjs]` `[create-sitecore-jss]` `[sitecore-jss-proxy]` `[sitecore-jss-angular]` Upgrade React to version 19 and Nextjs to version 15 ([#2078](https://github.com/Sitecore/jss/pull/2078)):
```

Comment:
Can you, please, add a separate entry for Angular and just mention cloudsdk upgrade? To make it visible